.editorconfig

```diff
@@ -6,19 +6,20 @@ root = true
 
 # Default settings:
 # A newline ending every file
-# Use 2 spaces as indentation
-# Trim trailing whitespace
+# Use 4 spaces as indentation
 [*]
 insert_final_newline = true
```

Comment:
Linux tools prefer final new line http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline